prov/gni/include/gnix_util.h

```diff
@@ -85,7 +85,7 @@ extern atomic_t gnix_debug_next_tid;
 				gnix_debug_pid = getpid();		\
 			}						\
 			snprintf(new_fmt, fmt_len, "[%%d:%%d] %s", fmt);	\
-			FI_LOG_FN(&gnix_prov, subsystem, new_fmt,		\
+			FI_LOG_FN(&gnix_prov, subsystem, new_fmt,	\
```

Comment:
I had added some debug here that was committed. When I removed the debug, this line was over 80 characters so the pre-commit check failed so I left it.